### PR TITLE
unarchive tool

### PR DESCRIPTION
**Pull request by peterbe**
Part of #198

This is the tool I used to create https://github.com/mdn/content/pull/515
It's quick-n-dirty and I'm not sure it's worth documenting it.

### CODE REVIEW

**Comment by peterbe** [comment]
@fiji-flo ping r?

**Comment by peterbe** [comment]
@chrisdavidmills I found the tool! It's a prototype here in this PR.

File: tool/cli.js [hunk 1]
```diff
@@ -419,6 +421,66 @@ program
...
+          "raw.html"
+        );
+        const rawHTML = fs.readFileSync(rawFilePath, "utf-8");
+        const created = Document.create(rawHTML, document.metadata);
+        console.log(`Created ${created}`);
+        countCreated++;
+        if (move) {
+          execGit(["rm", document.fileInfo.path], {}, CONTENT_ARCHIVED_ROOT);
+          execGit(["rm", rawFilePath], {}, CONTENT_ARCHIVED_ROOT);
+        }
```
Comment (fiji-flo) [inline tool/cli.js:480]: nit: That could be moved to `content/document.js` where `move`, `remove` and other document operations are found (maybe all of the belong in something separate even).

**Review by fiji-flo — APPROVED** [review]
👍  @peterbe I'm fine with merging it as is.

**Comment by chrisdavidmills** [comment]
Nice! Looking forward to getting to grips with this, and documenting it.

**Comment by peterbe** [comment]
@fiji-flo I moved most if the meat of that into `content/document.js` so now that the API contains both a `archive` and an `unarchive` function. Much neater than having all the business logic in the `tools/cli.js` file.

**Comment by peterbe** [comment]
Going to merge it since it's probably working (just last-minute refactor) and if it's not working anymore it's not critical or dangerous.

**Pull request merged**